### PR TITLE
T2533

### PR DESCRIPTION
**Pull request by cadecairos**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2533-header-is-too-tall-in-opera

### CODE REVIEW

**Comment by cadecairos** [comment]
I messed this up somehow...

**Comment by cadecairos** [comment]
There we go!

File: css/editors.less [hunk 1]
```diff
@@ -25,10 +25,11 @@
     padding: 0;
     width: 100%;
     list-style: none;
-    .clearfix();
 
-    > li {
+    > li.butter-editor-header-li {
```
Comment (k88hudson) [inline css/editors.less]: you can actually remove the `li` from this, i.e just  `.butter-editor-header-li`
also, just wondering -- what was the reason for this change?

File: css/normalize.less [hunk 2]
```diff
@@ -269,4 +274,12 @@ td {
 
 iframe {
   border: 0;
-}
\ No newline at end of file
+}
```
Comment (k88hudson) [inline css/normalize.less]: Remove one of these body declarations, https://github.com/mozilla/butter/pull/1225/files#L2R3

File: css/editors.less (hunk 1 above)
Comment (cadecairos) [inline css/editors.less]: to avoid the styles being applied to the last `<li>` in the group, which is used for hiding and showing the editor.

**Pull request merged**